packages/svelte/tsconfig.json

```diff
@@ -20,6 +20,6 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "preserveConstEnums": true,
-    "removeComments": true
+    "removeComments": false
```

Comment:
Needed to preserve the `/* @vite-ignore */` comment that prevents Vite from statically analyzing and bundling `svelte/server` during development.